public/locales/en/translation.json

```diff
@@ -52,7 +52,7 @@
     "edd-calculator": "https://www.edd.ca.gov/Unemployment/ui-calculator.htm",
     "edd-cares": "https://edd.ca.gov/about_edd/coronavirus-2019/cares-act.htm",
     "edd-pua": "https://edd.ca.gov/about_edd/coronavirus-2019/pandemic-unemployment-assistance.htm",
-    "edd-webapp": "https://portal.edd.ca.gov/WebApp/Login?resource_url=https://portal.edd.ca.gov/WebApp/Home",
+    "edd-webapp": "https://edd.ca.gov/Unemployment/UI_Online.htm",
```

Comment:
This link is shared with the guide! I think we need a new entry for the retro-certs link.

Reply:
oof good call